operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-statefulset.yaml

```diff
@@ -63,9 +73,9 @@ spec:
       {{- end }}
       {{- end }}
       nodeSelector:
-        {{- toYaml .Values.store_gateway.nodeSelector | nindent 8 }}
+        {{- $rolloutZone.nodeSelector | toYaml | nindent 8 }}
       affinity:
-        {{- toYaml .Values.store_gateway.affinity | nindent 8 }}
+        {{- $rolloutZone.affinity | toYaml | nindent 8 }}
```

Comment:
When last Krajo and I spoke, we weren't supporting migration from single to multi-zone or vice versa.

Reply:
Gotcha, that was probably true when you spoke to Krajo. The current plan is to support a live migration though. We (Grafana Labs) don't really have an option since this is the helm chart we ask our enterprise customers to use and we will need to provide an upgrade path for them in production.

It's probably more than twice as much work to include migration, so it makes sense to me if you'd rather a maintainer take over at that point.